lib/depends.ps1

```diff
@@ -46,17 +46,15 @@ function dep_resolve($app, $arch, $resolved, $unresolved) {
 }
 
 function runtime_deps($manifest) {
-    if($manifest.depends) { return $manifest.depends }
+    if ($manifest.depends) { return $manifest.depends }
 }
 
 function install_deps($manifest, $arch) {
     $deps = @()
 
-    if((requires_7zip $manifest $arch) -and !(7zip_installed)) {
-        $deps += '7zip'
-    }
-    if(requires_lessmsi $manifest $arch) { $deps += 'lessmsi' }
-    if($manifest.innosetup) { $deps += 'innounp' }
+    if (requires_7zip $manifest $arch) { $deps += '7zip' }
```

Comment:
I don't see a check for installed 7zip. Will this result into something like `7zip is already installed` being printed all the time when installing some manifest?

Reply:
https://github.com/lukesampson/scoop/blob/master/libexec/scoop-install.ps1#L107-L122

If `7zip` is deps, its installation status will not be reported.